doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -163,8 +163,8 @@ To identify that the KMU is used as a persistent storage backend for a specific
 | ``key lifetime`` (``psa_set_key_lifetime``)  | ``PSA_KEY_LIFETIME_FROM_PERSISTENCE_AND_LOCATION(persistence, location)``   | ``CRACEN_KEY_PERSISTENCE_REVOKABLE`` is a custom persistence mode, which will revoke the key slots when the key is destroyed.                      |
 |                                              | where persistence is set to one of the following:                           |                                                                                                                                                    |
 |                                              | ``PSA_KEY_PERSISTENCE_DEFAULT``,                                            | ``PSA_KEY_PERSISTENCE_DEFAULT`` should be used by applications that have no specific needs beyond what is met by implementation-specific features. |
-|                                              | ``PSA_KEY_PERSISTENCE_READ_ONLY``,                                          |                                                                                                                                                    |
-|                                              | ``CRACEN_KEY_PERSISTENCE_REVOKABLE``,                                       | ``PSA_KEY_PERSISTENCE_READ_ONLY`` is for read-only or write-once keys.                                                                             |
+|                                              | ``CRACEN_KEY_PERSISTENCE_READ_ONLY``,                                       |                                                                                                                                                    |
+|                                              | ``CRACEN_KEY_PERSISTENCE_REVOKABLE``,                                       | ``CRACEN_KEY_PERSISTENCE_READ_ONLY`` is for read-only or write-once keys.                                                                             |
```

Comment:
need to remove 3 spaces  for the last column, won't render otherwise

```suggestion
|                                              | ``CRACEN_KEY_PERSISTENCE_REVOKABLE``,                                       | ``CRACEN_KEY_PERSISTENCE_READ_ONLY`` is for read-only or write-once keys.                                                                          |
```